src/ui/wxWidgets/AddEditPropSheetDlg.cpp

```diff
@@ -982,8 +987,13 @@ void AddEditPropSheetDlg::InitAttachmentTab()
         DisableAttachmentControls();
```

Comment:
I'm wondering why we don't allow export of read-only entries. If we allowed it, then both r/o and protected cases could share the logic. If anything, it makes more sense to disable export for protected entries than read-only.